### ISSUE TITLE
Bug: `perfectionist/sort-enums` breaks eslint in version 3.1.0

### ISSUE THREAD

**Issue by hugoAlt** [azat-io/eslint-plugin-perfectionist#192]
### Describe the bug

The following code works in v3.0.0 and breaks eslint in v3.1.0

### Code example

```typescript
enum AnyEnum {
  A = 'A',
  B = 'B',
}
```

### ESLint version

8.57.0

### ESLint Plugin Perfectionist version

3.1.0

### Additional comments

```
TypeError: Cannot read properties of undefined (reading 'members')

Rule: "perfectionist/sort-enums"
    at TSEnumDeclaration ([X]\node_modules\eslint-plugin-perfectionist\dist\rules\sort-enums.js:79:19)

```
### Validations

- [X] Read the [docs](https://perfectionist.dev).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Thank you for your issue. I will fix this issue as soon as possible.

**Comment by azat-io** [comment]
Fixed in 47897646a4bc3fc2ad78c2105a8600931dc07c30. Released in [v3.1.1](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v3.1.1)

**Comment by brokenmass** [comment]
Same issue in 3.1.1 . The fix commit does not address the problem as it was not members that was undefined but body. 

getMembers should most likely be 

const getMembers = nodeValue => nodeValue.body?.members || nodeValue.members || [];

**Comment by azat-io** [comment]
Fixed in 3f3d77c6c3bef69495eb0c2897b79fa0353edf91. Released in [v3.1.2](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v3.1.2)
